packages/web-app-media-viewer/src/App.vue

```diff
@@ -201,8 +201,14 @@ export default {
 
     // keep a local history for this component
     window.addEventListener('popstate', this.handleLocalHistoryEvent)
+    const filePath = (filePath => {
```

Comment:
as discussed, please use chaining to make this a one-liner (which will need line breaks of course 😆 )